src/index.js

```diff
@@ -11,8 +12,18 @@ const kcscTheme = createMuiTheme({
       contrastText: "#6a0dad ",
     },
   },
+  typography: {
+    body1: {
+      fontSize: 24
+    },
+    button: {
+      fontSize: 24
+    }
+  }
 });
 
+axios.defaults.baseURL = process.env.REACT_APP_API_URL
```

Comment:
Maybe use the name `BACKEND_API_BASE_URL` instead of `REACT_APP_API_URL` just for clarity?

Reply:
Since we are in React we have to start with REACT_APP_..... So if so we could change to REACT_APP_BACKEND_BASE_URL :) But yeah i believe that is a better descriptive name 👍 Thx